src/main.cpp

```diff
@@ -161,11 +161,12 @@ namespace libsemigroups {
     init_dot(m);
     init_matrix(m);
     init_froidure_pin_base(m);
+    init_ukkonen(m);
 
 #ifdef VERSION_INFO
     m.attr("__version__") = VERSION_INFO;
 #else
-    m.attr("__version__") = "dev";
+    m.attr("__version__")                 = "dev";
```

Comment:
??

Reply:
This seems odd to me too, but I've double checked and this is indeed what `clang-format-15` wants. Should we add one of the "Don't format me please things"?